mm/kfence/naive.c

```diff
@@ -1,5 +1,4 @@
 // SPDX-License-Identifier: GPL-2.0
-//
```

Comment:
This piece is unrelated, but I don't mind.

Reply:
Yeah, I'm going to have some of these minor fixes all over the place. There is no point in creating separate commits -- we're not sending these patches to LKML. ;-)